### PR TITLE
Adding change to do incremental employee fetch from NetSuite

### PR DESCRIPTION
**Pull request by Shwetabhk**

### CODE REVIEW

File: apps/netsuite/connector.py [hunk 1]
```diff
@@ -647,7 +648,22 @@ def sync_employees(self):
         """
         subsidiary_mapping = SubsidiaryMapping.objects.get(workspace_id=self.workspace_id)
 
-        employees_generator = self.connection.employees.get_all_generator()
+        max_updated_at = DestinationAttribute.objects.filter(
+            workspace_id=self.workspace_id).all().aggregate(
```
Comment (ashwin1111) [inline apps/netsuite/connector.py:652]: add attribute type filter

File: apps/netsuite/connector.py [hunk 2]
```diff
@@ -647,7 +648,22 @@ def sync_employees(self):
...
+            max_updated_at=Max('updated_at')
+        )['max_updated_at']
+
+        if max_updated_at:
+            search_value = (max_updated_at - timedelta(days=30)).isoformat()
+
+            last_modified_date_query = {
```
Comment (ashwin1111) [inline apps/netsuite/connector.py:659]: handle new org case, last_modified_date_query is assigned inside if block

File: apps/netsuite/connector.py [hunk 3]
```diff
@@ -647,7 +648,22 @@ def sync_employees(self):
         """
         subsidiary_mapping = SubsidiaryMapping.objects.get(workspace_id=self.workspace_id)
 
-        employees_generator = self.connection.employees.get_all_generator()
+        max_updated_at = DestinationAttribute.objects.filter(
+            workspace_id=self.workspace_id).all().aggregate(
+            max_updated_at=Max('updated_at')
```
Comment (ashwin1111) [inline apps/netsuite/connector.py:653]: will it be updated_at=Max('updated_at') or max_column_name is the syntax?

Comment (Shwetabhk) [reply apps/netsuite/connector.py:653]: max_column_name is the syntax

**Pull request merged**